helix-term/src/lib.rs

```diff
@@ -47,3 +51,22 @@ fn filter_picker_entry(entry: &DirEntry, root: &Path, dedup_symlinks: bool) -> b
 
     true
 }
+
+/// Opens URL in external program.
+fn open_external_url_callback(
```

Comment:
**question**: any ideas about better location for this?

Reply:
This can be moved to helix-event once #8021 lands (will.need to be refactoring then anyway)